lighthouse-core/gather/fetcher.js

```diff
@@ -168,10 +164,16 @@ class Fetcher {
       timeoutHandle = setTimeout(() => reject(new Error(errorMessage)), timeout);
     });
 
-    return Promise.race([
+    const racePromise = Promise.race([
       timeoutPromise,
       requestInterceptionPromise,
     ]).finally(() => clearTimeout(timeoutHandle));
+
+    this.driver.evaluateAsync(`${injectIframe}(${JSON.stringify(url)})`, {
+      useIsolation: true,
+    }).catch(err => console.error(err));
```

Comment:
should we bother logging this?

Reply:
why not just put it into a `Promise.all([racePromise, injectPromise])` to get into the normal control flow?

Reply:
I don't see how. Send PR?